docs/app/Examples/collections/Breadcrumb/Content/BreadcrumbSectionPropExample.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import { Breadcrumb } from 'stardust'
+
+export default class BreadcrumbSectionPropExample extends Component {
+
+  sections = [
+    { text: 'Home', link: true },
+    { text: 'Search', active: true },
+  ];
```

Comment:
Same comment here regarding moving sections to the global scope.  Gasp!  Is that a semi colon 😝